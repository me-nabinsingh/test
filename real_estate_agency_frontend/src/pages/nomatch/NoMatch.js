import React from 'react';

const NoMatch = () => {
    return ( 
        <div className="album py-5 bg-light">
            <div className="px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center">
                <h1 className="display-4">404</h1>
                <p className="lead">Page you are looking doesn't exist. Please try different url.</p>
            </div>
        </div>
     );
}
 
export default NoMatch;