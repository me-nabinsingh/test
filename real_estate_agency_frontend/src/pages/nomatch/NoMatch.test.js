import React from 'react';
import NoMatch from './NoMatch';
import renderer from 'react-test-renderer';

describe('<NoMatch />', () => {
    it('renders without crashing', () => {
        const component = renderer.create(
            <NoMatch />
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});