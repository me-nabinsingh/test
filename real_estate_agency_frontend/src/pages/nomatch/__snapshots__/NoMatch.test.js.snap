// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> renders without crashing 1`] = `
<div
  className="album py-5 bg-light"
>
  <div
    className="px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center"
  >
    <h1
      className="display-4"
    >
      404
    </h1>
    <p
      className="lead"
    >
      Page you are looking doesn't exist. Please try different url.
    </p>
  </div>
</div>
`;
